src/ui/components/RatingsStatsPopover/index.tsx

```diff
@@ -6,6 +6,46 @@ import { helpers, toWorker } from "../../util";
 import ResponsivePopover from "../ResponsivePopover";
 import { PLAYER } from "../../../common";
 
+const PlayerNote = ({
+	note,
+	playerName,
+}: {
+	note: string | undefined;
+	playerName: string | undefined;
+}) => {
+	const [isOpen, setIsOpen] = useState(true);
+	if (!note) {
+		return null;
+	}
+	return (
+		<>
+			<div className="text-body-secondary bold">
+				{" "}
+				Notes {playerName !== undefined ? `for ${playerName}` : ""}
+			</div>
+			<div
+				className="text-wrap"
+				style={{
+					maxHeight: isOpen ? "10em" : "3em",
+					overflowY: "auto",
+				}}
+			>
+				{note}
+			</div>
+			<div className="d-flex justify-content-end">
+				{note.length > 100 && (
+					<button
+						className="btn btn-sm btn-light-bordered"
+						onClick={() => setIsOpen((prev) => !prev)}
+					>
+						{isOpen ? "less" : "more"}
+					</button>
+				)}
+			</div>
```

Comment:
I think the button is not needed, just show the whole note with some reasonable maxHeight. 10em might be fine, or slightly smaller.